### PR TITLE
DevTools 102 > Console > Live Expressions

### PR DESCRIPTION
**Pull request by sofiayem**
Documented https://developer.chrome.com/blog/new-in-devtools-102/#live-expression

Merge on May 24.

### CODE REVIEW

File: site/en/docs/devtools/console/live-expressions/index.md [hunk 1]
```diff
@@ -13,21 +14,38 @@ tags:
...
+
+To pin an expression to the top of the **Console**:
 
 1.  [Open the Console][1].
-2.  Click **Create Live Expression**
-    {% Img src="image/admin/AvEHQZdjwsrFJOcU8JLR.png", alt="Create Live Expression", width="33", height="23" %}.
-    The **Live Expression** text box appears.
+1.  Click {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/QVucttGri69wXWeE42qz.svg", alt="Create Live Expression.", width="20", height="20" %} **Create Live Expression**. The **Live Expression** text box appears.
+
+1. Type your expression in the text box. For example, you can use a live expression to [track element focus](docs/devtools/accessibility/focus/).
```
Comment (jecfish) [inline site/en/docs/devtools/console/live-expressions/index.md:27]: Probably check if we should update the keyboard shortcuts docs as well

**Review by jecfish — APPROVED** [review]
LGTM % some notes on shortcut documentation

File: site/en/docs/devtools/console/live-expressions/index.md [hunk 2]
```diff
@@ -3,6 +3,7 @@ layout: "layouts/doc-post.njk"
 title: "Watch JavaScript values in real-time with Live Expressions"
 authors:
   - kaycebasques
+  - sofiayem
 date: 2019-04-18
 #updated: YYYY-MM-DD
```
Comment (rachelandrew) [inline site/en/docs/devtools/console/live-expressions/index.md:8]:
```suggestion
updated: 2022-05-24
```

**Review by rachelandrew — APPROVED** [review]
LGTM - I just suggested adding the updated date.

**Pull request merged**